recipes/figanos/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "figanos" %}
+{% set version = "0.4.0" %}
+{% set python_min = "3.9" %}
```

Comment:
We only need to set `python_min` if the min version is higher than conda-forge's min. 

```suggestion
```